no-restricted-syntax.js

```diff
@@ -41,6 +41,12 @@ module.exports = [
 		selector:
 			"ImportDeclaration[source.value=classnames] ImportDefaultSpecifier[local.name!=/cx/]",
 	},
+	{
+		message:
+			"Instead of `<div onClick/>`, use <UnstyledButton/> (`button` element) or  <ClickableElement/> (accessible `div` element)",
```

Comment:
Also: updated description to suggest `UnstyledButton` as well (which is always preferred)

Reply:
The a11y plugin rules also apply on the app, so these 2 components help fix them correctly:

https://github.com/pixiebrix/pixiebrix-app/pull/5067/files#diff-a215210d34dc5b245c69ef42b4f3831552c05b2f10e1f80d110688a1df8e6ee0

The only relation to the extension is that it lives in the `@/components/*` folder. The app can use them freely, the other repos can import it I suppose.